internal/zinc-persist/src/main/scala/sbt/internal/inc/EnumSetSerializer.scala

```diff
@@ -0,0 +1,31 @@
+package sbt.internal.inc
+
+import java.util
+
+import scala.reflect.ClassTag
+
+case class EnumSetSerializer[E <: Enum[E]: ClassTag](allValues: Array[E]) {
```

Comment:
This is quite naive approach. I am open for suggestion about possible alternatives.